src/camera_positioner_table.cpp

```diff
@@ -149,11 +149,11 @@ class CameraPositioner {
           latest_detection_time = msg.detections[0].pose.header.stamp;
           get_tag0=true;
         }
-        if(msg.detections[i].id == table_tag_id_)
+        if(msg.detections[i].id[0] == table_tag_id_)
 	{
           get_tabletag=true;
-          tabletag_size=msg.detections[i].size;
-         tf::poseMsgToTF(msg.detections[i].pose.pose, tabletag_transform);
+          tabletag_size=msg.detections[i].size[0];
+         tf::poseMsgToTF(msg.detections[i].pose.pose.pose, tabletag_transform);
```

Comment:
whitespace error

Reply:
Unless you want to add a single commit that does full whitespace-fixup, please fix indentations whenever you touch a line.

Reply:
I fixed that. Both source files somehow have odd indentation. Should I add a commit with unified indentation to this pr?